gradle/libs.versions.toml

```diff
@@ -1,31 +1,28 @@
 [versions]
-agp = "8.0.2"
-coroutines = "1.8.0"
-dokka = "1.8.20"
-kotlin = "1.9.23"
-lifecycle = "2.6.1"
+agp = "8.10.0"
+coroutines = "1.10.2"
+dokka = "2.0.0"
+kotlin = "2.1.20"
```

Comment:
sdk should not force to use newer versions of the lib; it's a bad practice when sdk bump deps in the root app.
As I understand there is no reason for such updates

Reply:
downgraded down to 2.0.21